test/slack-send-test.js

```diff
@@ -61,13 +61,63 @@ describe('slack-send', () => {
         assert.equal(chatArgs.channel, 'C123456', 'Correct channel provided to postMessage');
         assert.equal(chatArgs.text, 'who let the dogs out?', 'Correct message provided to postMessage');
       });
+
+      it('should accept a payloadFilePath and use it\'s content in the message', async () => {
```

Comment:
Thanks for adding tests! I won't repeat but consistently renaming `payloadFilePath` would be appreciated.